tap_mambu/schemas/loan_accounts.json

```diff
@@ -390,11 +390,11 @@
             "properties": {
               "amount": {
                 "type": ["null", "number"],
-                "multipleOf": 1e-10
```

Comment:
can you change this:
```
"amount": {
                "type": ["null", "number"],
                "multipleOf": 1e-20
}
```
to look like this:
```
"amount": {
                "type": ["null", "string"],
                "format": singer.decimal
}
```
singer.decimal was built to handle more precision

can you do this for all of them